docs/containers-storage.conf.5.md

```diff
@@ -167,6 +167,14 @@ The `storage.options.overlay` table supports the following options:
 **ignore_chown_errors** = "false"
   ignore_chown_errors can be set to allow a non privileged user running with a  single UID within a user namespace to run containers. The user can pull and use any image even those with multiple uids.  Note multiple UIDs will be squashed down to the default uid in the container.  These images will have no separation between the users in the container. (default: false)
 
+**force_mask** = "0000|native|shared|private"
+  force_mask sets an octal permission mask and those bits are set for each new file or directory created in the storage.  The final permission for the new files and directories is set to ORIGINAL_PERMISSION|FORCE_MASK.  The values "native", "shared", "private" are also accepted.  (default: "native")
+  "native": it is equivalent to 0000.  The files maintain their original permission mask.
+  "private": it is equivalent to 0700.  The owner has rwx access to the files.
+  "shared": it is equivalent to 0755.  The owner has rwx access to the files and everyone else can read, access and execute them.
+  It is possible to use **force_mask** only when **mount_program** is specified.  It is an experimental feature.
```

Comment:
How about `Note: The force_mask Flag is an experimental feature, it could change in the future.`